src/MoviePreview.js

```diff
@@ -2,20 +2,33 @@ import React from 'react'
 import './MoviePreview.scss'
 
 const MoviePreview = (props) => {
-    console.log(props)
+  const { 
```

Comment:
Wow, that's a good destructuring, I still need to practice it to get better at it!